sympy/functions/elementary/exponential.py

```diff
@@ -596,7 +596,8 @@ def _eval_expand_log(self, deep=True, **hints):
 
     def _eval_simplify(self, ratio, measure):
         from sympy.simplify.simplify import expand_log, simplify
-        expr = self.func(simplify(self.args[0], ratio=ratio, measure=measure))
+        expr = self.func(simplify(self.args[0], ratio=ratio, measure=measure),
```

Comment:
I've checked this solution, it works! I think this is the best way to solve the problem.

What should I do with this pull request? Also I want to add  the following test to the
sympy/sympy/simplify/tests/test_simplify.py
```python
def test_issue_8866():
    assert simplify(log(x, 10, evaluate=False)) == simplify(log(x, 10))
```